README.md

```diff
@@ -137,6 +137,24 @@ channel.subscribe(events, new MessageListener() {
 });
 ```
 
+### Subscribing to a channel in delta mode ###
+
+Subscribing to a channel in delta mode enables delta compression. This is a way for a client to subscribe to a channel so that message payloads sent contain only the difference (ie the delta) between the present message and the previous message on the channel.
```

Comment:
```suggestion
Subscribing to a channel in delta mode enables [delta compression](https://www.ably.io/documentation/realtime/channels/channel-parameters/deltas). This is a way for a client to subscribe to a channel so that message payloads sent contain only the difference (ie the delta) between the present message and the previous message on the channel.
```